espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ComposeNodeElement.kt

```diff
@@ -56,7 +59,22 @@ class ComposeNodeElement(private val node: SemanticsNode) {
         get() = node.config.getOrNull(SemanticsProperties.Selected) == true
 
     val className: String
-        get() = node.config.getOrNull(SemanticsProperties.Role)?.toString() ?: COMPOSE_TAG_NAME
+        get() {
+            //  Compose API doesn't have class name info, as a workaround relaying on node SemanticsProperties.
+            val possibleClassProperties = listOf(
```

Comment:
can we move this list to a constant?

Reply:
Seems we can't non primitive types as constant, am I missing something?

`Const 'val' has type 'List<SemanticsPropertyKey<out Any>>'. Only primitives and String are allowed```

Reply:
we could make a [lazy](https://kotlinlang.org/docs/delegated-properties.html#standard-delegates) private property for it, so the value is only initialized once.